backend/main.py

```
@@ -0,0 +1,62 @@
+
+from app.api.v1 import api_router as api_router_v1
+from app.core.database import create_db_and_tables
+from app.core.settings import get_settings
+from fastapi import FastAPI
+from fastapi.middleware.cors import CORSMiddleware
+
+settings = get_settings()
+
+
+@asynccontextmanager
+async def lifespan(app: FastAPI):
```

Ruff (0.12.2)

Missing return type annotation for public function `lifespan`

(ANN201)